test/worlds/shapes.sdf

```diff
@@ -163,6 +163,28 @@
             <specular>0 0 1 1</specular>
           </material>
         </visual>
+
+        <sensor name="altimeter" type="altimeter">
```

Comment:
Can you put this in a new world?   The `shapes.sdf` is useful because it's such a simple world.  Adding a sensor makes it a bit more complicated, in my opinion.

Reply:
+1, there's also `altimeter.sdf` already that could be used.